synapse/federation/federation_server.py

```diff
@@ -597,6 +598,7 @@ async def on_send_join_request(
 
         time_now = self._clock.time_msec()
         return {
+            "event": event.get_pdu_json(),
```

Comment:
Should we only include this on certain room versions? Only if we've modified it?

Reply:
Maybe we should namespace it for now though, i.e. `org.matrix.msc3083.v2.event` or something?

Reply:
Sounds like a good idea.